force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +452,14 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        Integer age1 = person1.age;
+        Integer age2 = person2.age;
+        if(age1 > age2){
+            return person1;
+        }
+        else if (age1 == age2) {
+            return person1;
+        }
+        return person2; // Replace null with the variable you used to store the result
     }
```

Comment:
Logic is great. But I think you made it harder on yourself than it needed to be.  

if(person1.getAge() > person2.getAge() then return person1. Else return person2